backend/vertical-healthcare_ch/forum_datenaustausch_ch.invoice_rest-api/src/main/java/de/metas/vertical/healthcare/forum_datenaustausch_ch/rest/xml_to_olcands/XmlToOLCandsService.java

```diff
@@ -148,7 +148,7 @@ public JsonAttachment createOLCands(@NonNull final CreateOLCandsRequest createOL
 		final ResponseEntity<JsonAttachment> result = attachXmlToOLCandidates(
 				xmlInvoiceFile,
 				externalHeaderId,
-				Invoice440RequestConversionService.INVOICE_440_REQUEST_XSD); // store the XSD name so we don't need to extract it from the binary attachment data
+				Invoice450RequestConversionService.INVOICE_450_REQUEST_XSD); // store the XSD name so we don't need to extract it from the binary attachment data
```

Comment:
are you sure this is right?
i think this is about the input file's Version and since we get version 4.4 from the other system, i guess this needs to be 440..

Reply:
yup, reverted.